### ISSUE TITLE
help to implement 

### ISSUE THREAD

**Issue by brunocortes06** [Cykra/ng-password-meter#2]
I have this sample project: https://github.com/brunocortes06/db1-basic

Im trying to use you project to do the pass test but its not working, could you help?

The page is register.html the js file is hello.js

**Comment by nwocykra** [comment]
Yes sure! Once you have imported the js and css file you just need to add the following below your password input field:
`<ng-password-meter` `password="pw"></ng-password-meter>`

I saw in your code your model for the password input field is called pw and I specified accordingly.

**Comment by brunocortes06** [comment]
when i do this i get this "{{message}}" printed below the field

**Comment by nwocykra** [comment]
Are you getting any errors in your console? Looks the processing dies underway.

**Comment by brunocortes06** [comment]
yes i get an error about css:

SyntaxError: expected expression, got '.'
.pass-meter {

edit: fixed this message by declaring 
`	<link
	rel="stylesheet"
	 href="css/ng-password-meter.css">`

in the index.html

now i dont have any message com console, just the bars and the text {{message}}

![image](https://cloud.githubusercontent.com/assets/19157225/23936281/f4cc379a-092f-11e7-9e36-780b2e16da3c.png)
